core/tasks.py

```diff
@@ -495,6 +496,10 @@ def mkDefered(func, self=__undefinedFlag_, *args, **kwargs):
             response = taskClient.create_task(parent=parent, task=task)
 
             print('Created task {}'.format(response.name))
+            if conf.get("viur.tasks.logs",False):
+                logging.info("Create Task %s.%s" % (func.__name__, func.__module__))
+                logging.info("Create Task Arguments args = %s kwargs = %s"%(str(args),str(kwargs)))
```

Comment:
Please reduce this logging as well, and complete to the value of `env`:
```suggestion
                logging.info(f"Create task {func.__name__}.{func.__module__} with args={args!r} kwargs={kwargs!r} env={env!r}")
```